.travis.yml

```diff
@@ -25,5 +25,8 @@ env:
   - TEST_COMMAND='sh .travis.sigtest.sh'
   - TEST_COMMAND='mx/mx testdownstream'
 matrix:
-  allow_failures:
+    include:
+      - env: TEST_COMMAND='mx/mx -v build'
+        jdk: oraclejdk7
```

Comment:
I decided to use the [config suggested](https://github.com/jruby/jruby/blob/master/.travis.yml#L74-L75) by @eregon.